pkgs/by-name/ti/tigerjython/package.nix

```
@@ -0,0 +1,97 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  jre,
+}:
+let
+  icon = fetchurl {
+    # In case tigerjython/tjinstall becomes unavailable, use this url - see comment for src in MkDerivation
+    #url = "https://web.archive.org/web/20240623120114/https://raw.githubusercontent.com/tigerjython/tjinstall/master/tjlogo64.png";
+    url = "https://raw.githubusercontent.com/tigerjython/tjinstall/611c56d4e765731883656a5c4b71209d72b5ab74/tjlogo64.png";
+    hash = "sha256-tw3uDWLtcMHYmN6JGsEvVKLgI09v5DF27V2+OF9Z5tA=";
+  };
+in
+stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```
Doing this removes the C/C++ toolchain from being available in the build environment.
This *might* prevent some unnecessary rebuilds.
However, usually, if `gcc` changes, almost everything else gets rebuilt, so this probably doesn't help that much.